src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -326,7 +326,7 @@ protected function createFormBuilder($data = null, array $options = array()): Fo
     protected function getDoctrine(): ManagerRegistry
     {
         if (!$this->container->has('doctrine')) {
-            throw new \LogicException('The DoctrineBundle is not registered in your application.');
+            throw new \LogicException('The DoctrineBundle is not registered in your application. Try running "composer require orm"');
```

Comment:
I'd change this one. The error message mentions something about "DoctrineBundle", but then it tells you to install a strange "orm" thing. Could we change it to `Try running "composer require doctrine"`?